packages/tizen_app_manager/tizen/src/tizen_app_manager.h

```diff
@@ -21,12 +21,12 @@ class TizenAppManagerPlugin : public flutter::Plugin {
 
   virtual ~TizenAppManagerPlugin();
 
-  flutter::EncodableList m_applications;
-  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> m_launch_events;
+  flutter::EncodableList applications_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> launch_events_;
   std::unique_ptr<flutter::EventSink<flutter::EncodableValue>>
-      m_terminate_events;
-  bool m_registered_event_cb;
-  int m_registered_cnt;
+      terminate_events_;
+  bool registered_event_cb_;
+  int registered_cnt_;
```

Comment:
`applications_`, `registered_event_cb_` and `registered_cnt_` have been changed to private areas. However, `launch_events_` and `terminate_events_` have not been modified yet. Because `launch_events_` and `terminate_events_` are unique_ptr.

Reply:
> `launch_events_` and `terminate_events_` are unique_ptr

I'm sorry, but I don't understand this.
Would you like to refer to another plugin implementation?

Reply:
I think there was something I misunderstood. The other two variables have also been changed to private area.